stdlib/moreLabels.mli

```diff
@@ -194,5 +200,7 @@ module Set : sig
       val find_last_opt: f:(elt -> bool) -> t -> elt option
       val of_list: elt list -> t
     end
-  module Make : functor (Ord : OrderedType) -> S with type elt = Ord.t
+  module Make : functor (Ord : OrderedType) -> S
+    with type elt = Ord.t
+     and type t = Set.Make(Ord).t
```

Comment:
Well, it's already the case for the unlabeled variants. We are just saying that now if you create a set through `MoreLabels.Set`, it will be compatible with sets created through `Set`.

Some of these equalities where already exposed, for instance we already had MoreLabels.Hashtbl.t = Hashtbl.t`; it seems to that these were simply forgotten.